newsletter-november-2019.rst

```diff
@@ -0,0 +1,136 @@
+.. post:: November 04, 2019
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2019
+#########################################
+
+`It's...<https://www.youtube.com/watch?v=ydyXFUmv6S4>`__ not Monty Python's Flying Circus. Sorry. It is, however, the November edition of the newsletter!
+
+The `Sydney conference </conf/australia/2019/>`__ is coming up in just a few short weeks. It'll be 14th-15th November, and there are still a handful of tickets left. So if you were on the fence, get them now!
+
+Looking forward to next year, the site for `Portland 2020 </conf/portland/2019/>`__ is live, as is the `Call For Proposals </conf/portland/2020/cfp>`__. We'd love to hear all your awesome talk ideas, so get working on them! The CFP closes on 10th January, 2020.
```

Comment:
@plaindocs I'm currently planning to publish on Monday - think you'll be good by then? That PR seemed pretty close to being done when I glanced at it.

Reply:
Yeah,  I reckon.